HowTos/create_vpc.rst

```diff
@@ -1,39 +1,58 @@
 .. meta::
    :description: VPC Network CIDR Management Tool
-   :keywords: Aviatrix VPC Tracker, AWS VPC
+   :keywords: Aviatrix VPC Tracker, AWS VPC, AZURE VNet
 
 ###################################
 Create a VPC
 ###################################
 
-Use this tool to create a `VPC <https://www.aviatrix.com/learning/glossary/vpc.php>`_ that consists of a public subnet and a private subnet in two AZs and an IGW in the region and account of your choice. 
+Use this tool to create a `VPC <https://www.aviatrix.com/learning/glossary/vpc.php>`_ in AWS or a `VNet <https://a.aviatrix.com/learning/glossary/vnet.php>`_ in AZURE in the region and account of your choice. In addition, starting from 6.1, this tool creates multiple route tables associated with public and private subnets. One use case is to allow traffic load balancing when Aviatrix Spoke gateways are deployed. 
 
-The VPC CIDR range is from /16 to /24. The VPC name cannot contain underscore character ("_"). 
+To configure, go to Useful Tools at the main navigation on the left panel, select Create a VPC -> +Create.
 
-Go to Useful Tools at the main navigation on the left panel, select Create a VPC -> +Create.
+The VPC/VNet CIDR range is from /16 to /24.
 
-.. tip::
+Advanced
+---------
+
+When the "Advanced" option is selected, users are able to customize subnet size and number of pair of subnets (public subnet and private subnet). For example, entering 1 for Number of Availability Zones/Number of Subnets means that this tool will create 1 public subnet and 1 private subnet in the VPC/VNet.
+
+The VPC/VNet CIDR range is from /16 to /24.
 
-  Select the option Aviatrix Transit VPC if you deploy the AVX Transit Network. All necessary subnets and route tables will be fully populated. 
- 
 Aviatrix Transit VPC
 ----------------------
 
-When the "Aviatrix Transit VPC" option is selected, Create a VPC creates the following AWS VPC subnets.
+When the "Aviatrix Transit VPC" option is selected, all necessary subnets and route tables will be fully populated as below:
```

Comment:
Suggest a change to the word "below" to "follows".

..will be fully populated as follows: